source/includes/indexes/single-field.rb

```diff
@@ -0,0 +1,41 @@
+require 'mongo'
+
+# Replace the placeholders with your credentials
+uri = "<connection string>"
+
+# Set the server_api field of the options object to Stable API version 1
+options = { server_api: { version: "1" }}
+
+# Create a new client and connect to the server
+client = Mongo::Client.new(uri, options)
+
+# start-sample-data
+database = client.use('sample_mflix')
+collection = database[:movies]
+# end-sample-data
+
+# start-index-single
+# Create an index on the "title" field
```

Comment:
[n] I think some of the narrative comments can have the tense updated similar to the examples in [the code comment guide](https://docs.google.com/document/d/1Bnhh1OsGRUuXo_6yVUASGWiSX6sWuqrEFxPP6sLZDHM/edit?tab=t.0#heading=h.xv6glj2lnzjh).
```suggestion
# Creates an index on the "title" field
```